__tests__/integration.test.js

```diff
@@ -144,34 +151,86 @@ describe('/api/articles/:article_id/comments', () => {
-                    expect(typeof comment.created_at).toBe('string')
+                    expect(comment).toMatchObject(
+                        {
+                            comment_id: expect.any(Number),
+                            body: expect.any(String),
+                            article_id: expect.any(Number),
+                            author: expect.any(String),
+                            votes: expect.any(Number),
+                            created_at: expect.any(String),
+                        })
                 })
+
             });
     });
 
-    test.only("POST:/api/articles/:article_id/comments",()=>{
+    test("POST:/api/articles/:article_id/comments", () => {
+        return request(app)
+            .post('/api/articles/2/comments')
+            .send({
+                username: "rogersop",
+                body: "I love Poland"
+            })
+            .expect(201)
+            .then(({ body }) => {
+                const { newComment } = body
+                expect(newComment).toMatchObject(
+                    {
+                        comment_id: expect.any(Number),
+                        body: expect.any(String),
+                        article_id: expect.any(Number),
+                        author: expect.any(String),
+                        votes: expect.any(Number),
+                        created_at: expect.any(String),
+                    })
+
+            })
+
+
+    })
+    test("POST:/api/articles/200/comments Returns a 404 error if passed a non existing Id", () => {
+        return request(app)
+        .post('/api/articles/200/comments')
+        .send({
+            username: "rogersop",
+            body: "I love Poland"
+        })
+            .expect(404)
+            .then(({ body }) => {
+                const { message } = body
+                expect(message).toBe("The article id attempting to post in does not exist!")
+            })
+    })
+
+    test("POST:/api/articles/2/comments Returns a 404 error if passed a non existing Id", () => {
         return request(app)
         .post('/api/articles/2/comments')
         .send({
-            username : "rogersop",
-            body : "I love Poland"
+            username: "Dariusz",
+            body: "I love Poland"
         })
-        .expect(201)
-        .then(({body})=>{
-        
-        console.log(body)
-       
+            .expect(404)
+            .then(({ body }) => {
+                const { message } = body
+                expect(message).toBe("The username attempting to post does not exist!")
+            })
+    })
+    test("POST:/api/articles/NotValidEndpoint/comments Returns an empty array when passed an comment id without comments", () => {
+        return request(app)
+        .post('/api/articles/GarethBale/comments')
+        .send({
+            username: "rogersop",
+            body: "I love Poland"
         })
-
-
+            .expect(400)
```

Comment:
Some other tests we could add is for when an incomplete body is sent (e.g. no body in the request body object), or trying to post a comment to a valid but non-existent article id, and that any extra properties on the body are ignored